back/lib/participation_method/proposals.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
@kogre In general it might be good to look at the participation_methods, as they are very central in this PR.